src/main/java/pages/LoginPage.java

```diff
@@ -33,4 +39,96 @@ public String getPageTitle() {
         return driver.findElement(pageTitle).getText();
     }
 
-}
\ No newline at end of file
+    public boolean isTextPageTitleDisplayed() {
+        if (driver.findElement(pageTitle).isDisplayed()) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public String getRegisteredCustomerText() {
+        return driver.findElement(registeredCustomerText).getText();
+    }
+
+    public boolean isTextRegisteredCustomerDisplayed() {
```

Comment:
The same here - please to refer to comment for isTextPageTitleDisplayed() method.